AnomalyDetection/LeftDiscords/README.md

```diff
@@ -0,0 +1,9 @@
+## Left-discords Streaming Time-Series Anomaly Detection in Apache Flink
+
+A left-discord is a subsequence that is significantly dissimilar from all the sub sequences that precede it. In this sample, we demonstrate how to use the concept of left-discords to identify time-series anomalies in streams using Kinesis Data Analytics for Apache Flink.
```

Comment:
nit: First reference should use the marketing name "Amazon Kinesis Data Analytics". Nit because I am not sure how this applies to README